packs/newrelic/sensors/newrelic_application_sensor.py

```diff
@@ -240,6 +239,10 @@ def _dispatch_server_normal(self, payload, attempt_no=0):
             eventlet.spawn_after(self._normal_report_delay, self._dispatch_server_normal,
                                  payload, attempt_no + 1)
 
+    def _dispatch_trigger(self, trigger, payload):
+        self._sensor_service.dispatch(trigger, payload)
+        self._log.info('Dispatched %s with payload %s.', trigger, payload)
```

Comment:
Hm, I thought `self._sensor_service.dispatch` already logs it, but it looks like it doesn't. It would be a good idea to also log it there under debug log level.